doc/_extensions/zephyr/kconfig/__init__.py

```diff
@@ -91,7 +91,7 @@ def kconfig_load(app: Sphinx) -> Tuple[kconfiglib.Kconfig, Dict[str, str]]:
         root_args = argparse.Namespace(**{'soc_roots': [Path(ZEPHYR_BASE)]})
         v2_systems = list_hardware.find_v2_systems(root_args)
 
-        soc_folders = {soc.folder for soc in v2_systems.get_socs()}
+        soc_folders = {soc.folder[0] for soc in v2_systems.get_socs()}
```

Comment:
If board/SoC extensions will be allowed in-tree, will this have to be updated to include all folders?
```suggestion
        soc_folders = {folder for soc in v2_systems.get_socs() for folder in soc.folder}
```
Same with the occurrences of `board.dir[0]`.

Reply:
that depends on how we want to organize the docs in such case.

Will we want docs describing the in-tree soc / board to be located in a single folder, or do we want to allow documentation for a given soc / board to be placed at multiple locations.

I would like to leave such decision to doc maintainers and not as part of this PR.
@kartben thoughts ?

Reply:
OK, I would agree that the docs can be handled later, same with compliance.